ilastik/workflows/tracking/structured/structuredTrackingWorkflow.py

```diff
@@ -180,16 +180,17 @@ def connectLane(self, laneIndex):
         feature_dict_division[config.features_division_name] = { name: {} for name in config.division_features }
         opTrackingFeatureExtraction.FeatureNamesDivision.setValue(feature_dict_division)
 
-        opDivDetection.BinaryImages.connect( op5Binary.Output )
-        opDivDetection.RawImages.connect( op5Raw.Output )
-        opDivDetection.SegmentationImages.connect(opTrackingFeatureExtraction.LabelImage)
-        opDivDetection.ObjectFeatures.connect(opTrackingFeatureExtraction.RegionFeaturesAll)
-        opDivDetection.ComputedFeatureNames.connect(opTrackingFeatureExtraction.ComputedFeatureNamesAll)
-        opDivDetection.SelectedFeatures.setValue(configConservation.selectedFeaturesDiv)
-        opDivDetection.LabelNames.setValue(['Not Dividing', 'Dividing'])
-        opDivDetection.AllowDeleteLabels.setValue(False)
-        opDivDetection.AllowAddLabel.setValue(False)
-        opDivDetection.EnableLabelTransfer.setValue(False)
+        if self.divisionDetectionApplet:
+            opDivDetection.BinaryImages.connect( op5Binary.Output )
+            opDivDetection.RawImages.connect( op5Raw.Output )
+            opDivDetection.SegmentationImages.connect(opTrackingFeatureExtraction.LabelImage)
+            opDivDetection.ObjectFeatures.connect(opTrackingFeatureExtraction.RegionFeaturesAll)
+            opDivDetection.ComputedFeatureNames.connect(opTrackingFeatureExtraction.ComputedFeatureNamesAll)
+            opDivDetection.SelectedFeatures.setValue(configConservation.selectedFeaturesDiv)
+            opDivDetection.LabelNames.setValue(['Not Dividing', 'Dividing'])
+            opDivDetection.AllowDeleteLabels.setValue(False)
+            opDivDetection.AllowAddLabel.setValue(False)
+            opDivDetection.EnableLabelTransfer.setValue(False)
```

Comment:
Instead of calling `setValue()` here (in `connectLane()`) on level-0 slots  (`SelectedFeatures`, `LabelNames`, `AllowDeleteLabels`, `AllowAddLabel`, `EnableLabelTransfer`), I think it would be better to do that in `__init__()`, above.  The reason is that `connectLane()` is called repeatedly (for every image loaded), so you'll be calling `setValue()` with the same values over and over.

IIUC, the current code won't lead to wrong results, but if we ever decide to make these slots configurable by the user, then these lines will definitely need to be moved.  Otherwise, we'll be overwriting their settings every time they load a new image.